pkg/minikube/console/console.go

```diff
@@ -70,15 +70,19 @@ func HasStyle(style string) bool {
 
 // OutStyle writes a stylized and formatted message to stdout
 func OutStyle(style, format string, a ...interface{}) error {
-	OutStyle, err := applyStyle(style, useColor, fmt.Sprintf(format, a...))
+	outStyled, err := applyStyle(style, useColor, format, a...)
+	// format the output string and escape any outstanding '%' signs so that they don't
+	// get interpreted as a formatting directive down the line
+	outStyled = strings.Replace(outStyled, "%", "%%", -1)
```

Comment:
This should probably use `%` instead of "%"

Reply:
I think this is just a github thing, neither IDE nor gofmt complains